src/Discord.Net.WebSocket/Entities/Channels/SocketDMChannel.cs

```diff
@@ -51,13 +54,30 @@ public Task CloseAsync(RequestOptions options = null)
         /// <inheritdoc />
         public SocketMessage GetCachedMessage(ulong id)
             => _messages?.Get(id);
+        /// <summary>
+        ///     Gets the message associated with the passed <paramref name="id"/>.
+        /// </summary>
+        /// <param name="id">The id of the message you want to retrieve</param>
```

Comment:
* Use `snowflake identifier` instead of `id`.
* Use neutral subject or remove the subject entirely (e.g. `The snowflake identifier of the message to retrieve`)
* Missing full-stop punctuation.